header:
    seq: 1
    stamp: "now"
height: 480
width: 640
distortion_model: "plumb_bob"
D: [-0.364865, 0.111417, -0.003288, 0.001529, 0.000000]
K: [ 443.84389,    0.     ,  300.09341,
       0.     ,  448.89312,  259.5822 ,
       0.     ,    0.     ,    1.     ]
R: [ 1.,  0.,  0.,
          0.,  1.,  0.,
          0.,  0.,  1.]
P: [ 339.26248,    0.     ,  290.83801,    0.     ,
       0.     ,  394.3075 ,  263.45594,    0.     ,
       0.     ,    0.     ,    1.     ,    0.     ]
binning_x: 0
binning_y: 0
roi:
    x_offset: 0
    y_offset: 0
    height: 0
    width: 0
    do_rectify: False
